source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -143,7 +143,9 @@ public PimsAcquisitionFile Update(PimsAcquisitionFile acquisitionFile)
             }
 
             this.Context.Entry(existingAcqFile).CurrentValues.SetValues(acquisitionFile);
-            this.Context.UpdateChild<PimsAcquisitionFile, long, PimsAcquisitionFilePerson>(p => p.PimsAcquisitionFilePeople, acquisitionFile.Internal_Id, acquisitionFile.PimsAcquisitionFilePeople.ToArray());
+            this.Context.UpdateChild<PimsAcquisitionFile, long, PimsAcquisitionFilePerson>(p => p.PimsAcquisitionFilePeople, acquisitionFile.Internal_Id, acquisitionFile.PimsAcquisitionFilePeople.ToArray(), true);
+            this.Context.UpdateChild<PimsAcquisitionFile, long, PimsAcquisitionOwner>(p => p.PimsAcquisitionOwners, acquisitionFile.Internal_Id, acquisitionFile.PimsAcquisitionOwners.ToArray(), true);
```

Comment:
In this case passing the true flag makes since since the ref to the owner may need to be updated as well as the owner itself.

Reply:
Updated